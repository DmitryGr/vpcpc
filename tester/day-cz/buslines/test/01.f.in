10
5 3
7 8
5 10
8 1
5 2
1 2
10 9
4 3
6 10
