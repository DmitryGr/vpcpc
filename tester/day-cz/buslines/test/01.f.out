23
27
17
9
35
9
9
17
9
24
